conbench/templates/compare-list.html

```diff
@@ -162,15 +147,15 @@ <h6>Top Outliers</h6>
                   {{ c.analysis.pairwise.percent_change }} %
                 </td>
               {% else %}
-                <td></td>
+                <td data-order="99999999">n/a</td>
               {% endif %}
-              {% if c.analysis.lookback_z_score.z_score is not none %}
```

Comment:
This piece of code assumed that `if c.analysis.lookback_z_score` is never `None` (wrong assumption).